Makefile

```diff
@@ -162,3 +162,8 @@ generate-migrations:
 .PHONY: tailwindcss
 tailwindcss:
 	npx --yes tailwindcss build -o web/static/tailwind.css
+
+.PHONY: web
+web:
+	rm -fR web/webplatform/dist
+	cd web/webplatform && unzip autokitteh-web-*.zip
```

Comment:
- how/where is autokitteh-web-*.zip is created?
- doc/explanation how to get it
- check that it's present and error message before trying to extract?

Reply:
@itayd - why do we need/want all webplatform assets (e.g. purr etc) in autokitteh repo?

Reply:
@ig-ra we need those assets bundled in the webplatform because we use it there to create projects from templates, and these samples are the base of it (manifest and files).